contentcuration/contentcuration/frontend/shared/views/LanguageDropdown.vue

```diff
@@ -106,7 +106,7 @@
     },
     methods: {
       languageText(item) {
-        return this.$tr('languageItemText', { language: item.native_name, code: item.id });
+        return this.$tr('languageItemText', { language: item.first_native_name, code: item.id });
```

Comment:
Hi @cerdo0! It seems like the value of ` item.first_native_name ` is undefined. The output now only shows the code of the language in the dropdown.

Reply:
may be something like this could work?
`  const firstNativeName = item.native_name.split(',')[0].trim()`
cc @MisRob?

Reply:
https://github.com/learningequality/studio/issues/2074 is a pretty old issue, so yes, it seems `first_native_name` is obsolete. I don't see it anywhere in the codebase https://github.com/search?q=repo%3Alearningequality%2Fstudio+first_native_name&type=code.

@ozer550 your suggestion makes sense. I'd just suggest confirming that
> native_name can be a comma separated list of names

is still a valid remark.

Reply:
Hey @MisRob @ozer550 ,
I have made the changes as suggested by you guys. Please review my changes.